tsaqafah.csl

```diff
@@ -0,0 +1,501 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="text" et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" initialize="false" initialize-with="." page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Tsaqafah</title>
+    <title-short>tsaqafah</title-short>
+    <id>http://www.zotero.org/styles/tsaqafah</id>
+    <link href="http://www.zotero.org/styles/tsaqafah" rel="self"/>
+    <link href="http://ejournal.unida.gontor.ac.id/index.php/tsaqafah/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <email>lpp@unida.gontor.ac.id</email>
+      <name>Tim LPP</name>
+    </author>
+    <contributor>
+      <name>Lalu Ganda</name>
+      <email>laluganda@unida.gontor.ac.id</email>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="theology"/>
+    <issn>1411-0334</issn>
+    <eissn>2460-0008</eissn>
+    <summary>note with bibliography (adapted-modified from jurnal al-jami'ah)</summary>
```

Comment:
`<summary>note with bibliography (adapted-modified from jurnal al-jami'ah)</summary>`

@daredmayo, does this mean that there is a CSL style for the journal al-jami'ah? Do you know where that style is available? (I don't think we have it in our repository)

Reply:
yes, there is CSL styl for journal al-jami'ah...
i found it here http://aljamiah.or.id/support/others/al-jamiah-journal-of-islamic-studies-indonesia.csl

Reply:
Could you please ask the editor of that journal (at editor@aljamiah.org) whether they are okay with you submitting a modified version here under our required "Creative Commons Attribution-ShareAlike 3.0 License" license?

http://aljamiah.or.id/support/others/al-jamiah-journal-of-islamic-studies-indonesia.csl doesn't have any licensing information, so we rather make sure.

Reply:
Thank you, i will contact that editor to make sure for the license

Reply:
By the way, we'd also be happy to add their style to our repository, if they're interested in that.

Reply:
i have contact Al-Jami'ah editor and they have no problem about the licence

Reply:
Great, thanks for checking.